docker-compose.yml

```diff
@@ -54,9 +54,11 @@ services:
       LANGFUSE_TIMEOUT: "10"
       LANGFUSE_SAMPLE_RATE: "1.0"
       OTEL_EXPORTER_TYPE: "console" # or "console" # Define the exporter type (console/otlp)
-      # OTEL_OTLP_ENDPOINT: "http://jaeger:4317" # Define the endpoint for the otlp exporter. Uncomment this line if you want to use jaeger.
+      # OTEL_OTLP_ENDPOINT: "http://otel-collector:4317" # Define the endpoint for the otlp exporter.
```

Comment:
Worth mentioning in the comment that this is in fact GRPC? Tripped me up the first time but I suppose the port is clear 👍

Just in terms of GRPC/HTTP support.

Reply:
I added a new variable, `OTEL_EXPORTER_PROTOCOL,` and said HTTP support and GRPC.

Reply:
Amazing, thanks! Sounds like the most versatile approach.